velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -224,6 +278,84 @@ class StreamingAggregationTest : public OperatorTestBase {
 
     EXPECT_EQ(NonPODInt64::constructed, NonPODInt64::destructed);
   }
+
+  void testMultiKeyDistinctAggregation(
```

Comment:
This tests doesn't seem to use streaming aggregation. Is this intentional?

Reply:
No, Fixed. I should use `PlanBuilder::streamingAggregation` explicitly and use `AggregationNode::Step::kSingle`.